pkgs/by-name/ca/caido/package.nix

```
@@ -0,0 +1,39 @@
+{ lib, fetchurl, appimageTools, makeWrapper }:
```

```suggestion
{ lib
, fetchurl
, appimageTools
, makeWrapper
}:
```
This fits better the style of nixpkgs.